Documentation/ceph-object-store-crd.md

```diff
@@ -89,7 +89,7 @@ The gateway settings correspond to the RGW daemon settings.
 
 * `type`: `S3` is supported
 * `sslCertificateRef`: If the certificate is not specified, SSL will not be configured. If specified, this is the name of the Kubernetes secret that contains the SSL certificate to be used for secure connections to the object store. Rook will look in the secret provided at the `cert` key name. The value of the `cert` key must be in the format expected by the [RGW service](https://docs.ceph.com/docs/master/install/ceph-deploy/install-ceph-gateway/#using-ssl-with-civetweb): "The server key, server certificate, and any other CA or intermediate certificates be supplied in one file. Each of these items must be in pem form."
-* `port`: The port on which the RGW pods and the RGW service will be listening (not encrypted).
+* `port`: The port on which the RGW pods and the RGW service will be listening (not encrypted). This will be an internal port for the RGW daemon only if host network is enabled. Otherwise it uses 8080 by default.
```

Comment:
```suggestion
* `port`: The port on which the RGW pods and the RGW service will be listening (not encrypted) if host networking is enabled. Otherwise, this will be the Service port and the internal port for the RGW daemon will be hardcoded to 8080.
```